src/api/app/views/webui/user/_basic_info.html.haml

```diff
@@ -42,7 +42,7 @@
         = link_to('#', id: 'toggle-in-place-editing', title: 'Edit Your Account') do
           %i.fas.fa-user-edit
           %span.nav-item-name Edit
-    - if policy(user).update?
+    - if policy(user).update? && User.session
```

Comment:
This solution fixes the issue. However, this means that the `_nobody_` user has permissions to update itself?

Shouldn't we go for a solution which modifies the User update policy instead, to prevent the `_nobody_` user from updating itself?

Reply:
Yes, you are right. I've changed the policy instead. Let's see...